xcm-simulator/src/storagehub/configs/mod.rs

```diff
@@ -346,6 +348,13 @@ parameter_types! {
     pub const MetadataDepositPerByte: Balance = 1 * UNIT;
 }
 
+impl pallet_parameters::Config for Runtime {
```

Comment:
Did you add pallet parameters just to match what was in the actual runtime?

Reply:
Yes, I want to make sure the runtime we use for the XCM simulator's tests is as close as possible to the actual runtime